plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/thrift/ThriftHiveMetastoreClient.java

```diff
@@ -177,6 +180,36 @@ public Database getDatabase(String dbName)
         return client.getDatabase(dbName);
     }
 
+    @Override
+    public Map<String, List<String>> getAllTables()
+            throws TException
+    {
+        return parseTableMetadata(client.getTableMeta("*", "*", ImmutableList.of()));
+    }
+
+    @Override
+    public Map<String, List<String>> getAllViews()
+            throws TException
+    {
+        return parseTableMetadata(client.getTableMeta("*", "*", ImmutableList.of("VIEW")));
+    }
+
+    private static Map<String, List<String>> parseTableMetadata(List<TableMeta> meta)
```

Comment:
tableMeta

Reply:
I changed it to `tablesMetadata` to differentiate from local variable `tableMetadata`